src/generated/resources/assets/railways/lang/zh_cn.json

```
@@ -0,0 +1,91 @@
+
+	"entity.railways.benchcart": "工作台矿车",
+	"entity.railways.conductor": "指挥机器人",
+	"entity.railways.jukeboxcart": "唱片机矿车",
+
+	"item.railways.benchcart": "工作台矿车",
+	"item.railways.black_conductor_cap": "黑色列车长帽",
+	"item.railways.black_incomplete_conductor_cap": "黑色列车长帽（半成品）",
+	"item.railways.blue_conductor_cap": "蓝色列车长帽",
+	"item.railways.blue_incomplete_conductor_cap": "蓝色列车长帽（半成品）",
+	"item.railways.brown_conductor_cap": "棕色列车长帽",
+	"item.railways.brown_incomplete_conductor_cap": "棕色列车长帽（半成品）",
+	"item.railways.cyan_conductor_cap": "青色列车长帽",
+	"item.railways.cyan_incomplete_conductor_cap": "青色列车长帽（半成品）",
+	"item.railways.gray_conductor_cap": "灰色列车长帽",
+	"item.railways.gray_incomplete_conductor_cap": "灰色列车长帽（半成品）",
+	"item.railways.green_conductor_cap": "绿色列车长帽",
+	"item.railways.green_incomplete_conductor_cap": "绿色列车长帽（半成品）",
+	"item.railways.jukeboxcart": "唱片机矿车",
+	"item.railways.light_blue_conductor_cap": "淡蓝色列车长帽",
+	"item.railways.light_blue_incomplete_conductor_cap": "淡蓝色列车长帽（半成品）",
+	"item.railways.light_gray_conductor_cap": "淡灰色列车长帽",
+	"item.railways.light_gray_incomplete_conductor_cap": "淡灰色列车长帽（半成品）",
+	"item.railways.lime_conductor_cap": "黄绿色列车长帽",
+	"item.railways.lime_incomplete_conductor_cap": "黄绿色列车长帽（半成品）",
+	"item.railways.magenta_conductor_cap": "品红色列车长帽",
+	"item.railways.magenta_incomplete_conductor_cap": "品红色列车长帽（半成品）",
+	"item.railways.orange_conductor_cap": "橙色列车长帽",
+	"item.railways.orange_incomplete_conductor_cap": "橙色列车长帽（半成品）",
+	"item.railways.pink_conductor_cap": "粉红色列车长帽",
+	"item.railways.pink_incomplete_conductor_cap": "粉红色列车长帽（半成品）",
+	"item.railways.purple_conductor_cap": "紫色列车长帽",
+	"item.railways.purple_incomplete_conductor_cap": "紫色列车长帽（半成品）",
+	"item.railways.red_conductor_cap": "红色列车长帽",
+	"item.railways.red_incomplete_conductor_cap": "红色列车长帽（半成品）",
+	"item.railways.track_incomplete_acacia": "金合欢木列车轨道（半成品）",
+	"item.railways.track_incomplete_birch": "白桦木列车轨道（半成品）",
+	"item.railways.track_incomplete_crimson": "绯红木列车轨道（半成品）",
+	"item.railways.track_incomplete_dark_oak": "深色橡木列车轨道（半成品）",
+	"item.railways.track_incomplete_jungle": "丛林木列车轨道（半成品）",
+	"item.railways.track_incomplete_oak": "橡木列车轨道（半成品）",
+	"item.railways.track_incomplete_spruce": "云杉木列车轨道（半成品）",
+	"item.railways.track_incomplete_warped": "诡异木列车轨道（半成品）",
+	"item.railways.white_conductor_cap": "白色列车长帽",
+	"item.railways.white_incomplete_conductor_cap": "白色列车长帽（半成品）",
+	"item.railways.yellow_conductor_cap": "黄色列车长帽",
+	"item.railways.yellow_incomplete_conductor_cap": "黄色列车长帽（半成品）",
+
+	"itemGroup.railways": "机械动力：汽鸣铁道",
+
+	"railways.ponder.conductor_constructing.header": "创建一个指挥机器人",
+	"railways.ponder.conductor_constructing.text_1": "指挥机器人由安山机壳组成。",
+	"railways.ponder.conductor_redstoning.header": "借助指挥机器人远程操控红石元件",
+	"railways.ponder.conductor_redstoning.text_1": "指挥机器人可以在玩家注视它时按下按钮。",
+	"railways.ponder.conductor_redstoning.text_2": "它同样可以在玩家注视它时拉动拉杆。",
+	"railways.ponder.conductor_redstoning.text_3": "当玩家戴着与其颜色不一致的列车长帽时，指挥机器人不会激活红石元件。",
```

```suggestion
	"railways.ponder.conductor_constructing.text_1": "指挥机器人由安山机壳组成",
	"railways.ponder.conductor_redstoning.header": "借助指挥机器人远程操控红石元件",
	"railways.ponder.conductor_redstoning.text_1": "指挥机器人可以在玩家注视它时按下按钮",
	"railways.ponder.conductor_redstoning.text_2": "它同样可以在玩家注视它时拉动拉杆",
	"railways.ponder.conductor_redstoning.text_3": "当玩家戴着与其颜色不一致的列车长帽时，指挥机器人不会激活红石元件",
```
The original translation for Create has no periods in ponder lines.